BE/Cloud/KMS.Product.Ktm.Api/Controllers/ReportController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using AutoMapper;
+using KMS.Product.Ktm.Api.Exceptions;
+using KMS.Product.Ktm.Entities.DTO;
+using KMS.Product.Ktm.Services.KudoService;
+
+namespace KMS.Product.Ktm.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ReportController : ControllerBase
+    {
+
+        private readonly IKudoService _kudoService;
+        private readonly IMapper _mapper;
+
+        /// <summary>
+        /// Inject Kudo service
+        /// </summary>
+        /// <returns></returns>
+        public ReportController(IKudoService kudoService, IMapper mapper)
+        {
+            _kudoService = kudoService ?? throw new ArgumentNullException($"{nameof(kudoService)}");
+            _mapper = mapper;
+        }
+
+        /// <summary>
+        /// Get all kudos
+        /// GET: api/Kudo
+        /// </summary>
+        /// <returns>
+        /// Success: returns 200 status code with a collection of all kudos        
+        /// Failure: returns 500 status code with an exception message
+        /// </returns>
+        [HttpGet]
+        public async Task<IActionResult> GetKudosForReport(
+            [FromQuery] DateTime? dateFrom, 
```

Comment:
We also need another report: summary report.
In that report, user only want to know how many kudos were created (we don't need detail about each kudos).
Something like: From 01/01/2020 to 20/02/2020 
- There are 20 kudos were created in team 1
- There are 30 kudos were created in team 2

Or at the team level, we need reports: each member create/receive how many kudos

We also need a filter by person who create the kudos